dev/ci/ci-common.sh

```diff
@@ -19,26 +22,19 @@ then
 elif [ -d "$PWD/_build_vo/" ];
 then
     # Dune Ocaml build, vo build using make
-    export OCAMLPATH="$PWD/_build/install/default/lib/:$PWD/_install_ci/lib:$OCAMLPATH"
-    export COQBIN="$PWD/_build/install/default/bin"
-    export COQLIB="$PWD/_build_vo/default/"
-    # We set this due to dune + vo make mode not creating lib/coq anymore
-    export COQCORELIB="$PWD/_build/install/default/lib/coq-core"
+    export OCAMLPATH="$PWD/_build_vo/default/lib/:$OCAMLPATH"
+    export COQBIN="$PWD/_build_vo/default/bin"
+    export COQLIB="$PWD/_build_vo/default/lib/coq"
+    export COQCORELIB="$PWD/_build_vo/default/lib/coq-core"
     CI_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
     export CI_BRANCH
 elif [ -d "$PWD/_build/install/default/" ];
 then
     # Full Dune build
-    export OCAMLPATH="$PWD/_build/install/default/lib/:$PWD/_install_ci/lib:$OCAMLPATH"
```

Comment:
Why is it not needed anymore to extend `OCAMLPATH` with `_install_ci/lib`?

Reply:
Not sure why it was needed in the first place, that directory is only relevant in the first case, right?

Reply:
It comes from https://github.com/coq/coq/pull/13464

Reply:
>It comes from #13464

Then, it should probably be kept.

>that directory is only relevant in the first case, right?

I don't understand what you mean by this.

Reply:
I mean that we only need to add _install_ci to the path when we are running in the CI, that is to say, the first if branch.

I'm not sure why #13464 did modify these parts which IIUC are not taken during CI, only during "manual" `make ci-foo` calls.

Reply:
What happens if you run `make ci-compcert` locally? How does it find its dependencies?

Reply:
I'm not sure what is going on, but in this case there will be nothing in `_install_ci` , but things are installed in by the ci targets in whatever prefix we setup at configure time.

Reply:
https://github.com/coq/coq/blob/6bd39f29e675e125edfcb1dcb7a0fc5997049741/dev/ci/ci-menhir.sh#L6

Reply:
I see, thanks. That doesn't feel right at all tho.

Reply:
Why doesn't it?

Reply:
It feels weird to mess manually with the OCaml setup, usually if using opam you would pin the package to the right tree and let the standard tooling take care, or you could vendor that.

As of today we manage opam CI deps in the docker image, so indeed we should somehow extend that mechanism, but then Nix people etc... I'm not sure where they land.

Reply:
In any case, the right path for this, for example in a dune build, would be `_build/install/default/lib/menhir` etc....

Reply:
Any local install is fine with me, but no global one (precisely because I'm a NixOS user).